Auto update failing for rlwrap

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [rlwrap](https://github.com/termux/termux-packages/blob/4c3c84ad8e112bf5d77d700fe94cf2ddadf2e786/packages/rlwrap) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating rlwrap [Current version: 0.46.2]
INFO: package being updated to 0.47.
Downloading https://fossies.org/linux/privat/rlwrap-0.47.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  211k    0  211k    0     0   145k      0 --:--:--  0:00:01 --:--:--  145k
100  247k    0  247k    0     0   169k      0 --:--:--  0:00:01 --:--:--  169k

You are about to commit these changes:

--------------------
diff --git a/packages/rlwrap/build.sh b/packages/rlwrap/build.sh
index 7c80630abf..db5a725b37 100644
--- a/packages/rlwrap/build.sh
+++ b/packages/rlwrap/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/hanslub42/rlwrap
 TERMUX_PKG_DESCRIPTION="Wrapper using readline to enable editing of keyboard input for commands"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.46.2"
+TERMUX_PKG_VERSION="0.47"
 TERMUX_PKG_SRCURL=https://fossies.org/linux/privat/rlwrap-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=de206788a8179a6bee960956d96ae065975efd1cf5b6b365bbea5b0e610af1a7
+TERMUX_PKG_SHA256=6907089dbf1640159dba7cb470c949699c27e35ddac590cc7d4c6f51cdfa1c4f
 TERMUX_PKG_DEPENDS="ncurses, readline"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/rlwrap): 0.47


Not committing to Git!

INFO: Trying to build package.
INFO: Package rlwrap exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building rlwrap for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  35092      0 --:--:-- --:--:-- --:--:-- 35117
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  32755      0 --:--:-- --:--:-- --:--:-- 32800
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13319  100 13319    0     0  34045      0 --:--:-- --:--:-- --:--:-- 34063
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   2215      0 --:--:-- --:--:-- --:--:--  2215
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   2215      0 --:--:-- --:--:-- --:--:--  2215
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Fri 22 Aug 2025 12:12:03 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   2270      0 --:--:-- --:--:-- --:--:--  2269
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Thu 21 Aug 2025 01:57:20 PM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   2271      0 --:--:-- --:--:-- --:--:--  2269
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   2271      0 --:--:-- --:--:-- --:--:--  2269
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Fri 08 Aug 2025 06:40:59 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  525k  100  525k    0     0   767k      0 --:--:-- --:--:-- --:--:--  766k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87728  100 87728    0     0   233k      0 --:--:-- --:--:-- --:--:--  232k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32964  100 32964    0     0   112k      0 --:--:-- --:--:-- --:--:--  113k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 31 39644   31 12303    0     0  19403      0  0:00:02 --:--:--  0:00:02 19405
100 39644  100 39644    0     0  52301      0 --:--:-- --:--:-- --:--:-- 52300
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-support/libandroid-support_29_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10852  100 10852    0     0  29221      0 --:--:-- --:--:-- --:--:-- 29250
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.3.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/readline/readline_8.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  286k  100  286k    0     0   590k      0 --:--:-- --:--:-- --:--:--  590k
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading rlwrap@0.47 source from 'https://fossies.org/linux/privat/rlwrap-0.47.tar.xz' if necessary...
Downloading https://fossies.org/linux/privat/rlwrap-0.47.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  247k    0  247k    0     0   202k      0 --:--:--  0:00:01 --:--:--  202k
Applying patch: disable-bracketed-paste.patch
Applying patch: fix-hardcoded-paths.patch
Applying patch: ptytty.c.patch
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /usr/bin/autoconf --force
autoreconf: running: /usr/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
autoreconf: Leaving directory '.'
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
checking build system type... x86_64-pc-linux-gnu
checking host system type... aarch64-unknown-linux-android
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make sets $(MAKE)... (cached) yes
checking whether build environment is sane... yes
checking whether make supports the include directive... yes (GNU style)
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C11 features... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking dependency style of aarch64-linux-android-clang... none
checking for ptytty_create in -lptytty... no
configure: error: libptytty not found, install it or specify  --without-libptytty
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-08-22 01:23:41 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/17142134137">17142134137</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/rlwrap/build.sh
+++ b/packages/rlwrap/build.sh
@@ -2,13 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://github.com/hanslub42/rlwrap
 TERMUX_PKG_DESCRIPTION="Wrapper using readline to enable editing of keyboard input for commands"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.46.2"
-TERMUX_PKG_SRCURL=https://fossies.org/linux/privat/rlwrap-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=de206788a8179a6bee960956d96ae065975efd1cf5b6b365bbea5b0e610af1a7
+TERMUX_PKG_VERSION="0.47"
+TERMUX_PKG_SRCURL=https://github.com/hanslub42/rlwrap/releases/download/v${TERMUX_PKG_VERSION}/rlwrap-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=225e2c64023336585cd50e7ff4ec425f84664ceaae349ddf51e56139364e3130
 TERMUX_PKG_DEPENDS="ncurses, readline"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--without-libptytty
 ac_cv_func_grantpt=yes
 ac_cv_func_unlockpt=yes
 ac_cv_lib_util_openpty=no

